### PR TITLE
Event.java: add location and date

### LINKED ISSUE #134: Add Date and Location to Events Module
Date is LocalDateTime
Location will be a string.

### PR DESCRIPTION
**Pull request by jieffyy**
Adds the following to Event:
- Location
- Event Date

Storage to json has been updated for these fields.

The UI has not been updated yet.

image

### CODE REVIEW

File: src/main/java/seedu/address/model/event/EventDate.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    /**
+     * Returns true if a given string is a valid date.
+     */
+    public static boolean isValidEventDate(String test) {
+        requireNonNull(test);
+        try {
+            LocalDateTime.parse(test, VALIDATION_FORMATTER);
+            return true;
+        } catch (Exception e) {
```
Comment (yanchenglee98) [inline src/main/java/seedu/address/model/event/EventDate.java]: Should catch for `DateTimeParseException` instead of just `Exception`

**Review by yanchenglee98 — COMMENTED** [review]
Good quality work. Just a minor nit

File: src/main/java/seedu/address/model/event/EventDate.java (hunk 1 above)
Comment (jieffyy) [reply src/main/java/seedu/address/model/event/EventDate.java]: Sure

**Review by yanchenglee98 — APPROVED** [review]
LGTM

**Pull request merged**